solr/core/src/test/org/apache/solr/search/TestFilteredDocIdSet.java

```diff
@@ -133,24 +133,8 @@ public void testNullDocIdSet() throws Exception {
     // Now search w/ a Query which returns a null Scorer
     DocSetQuery f = new DocSetQuery(null) {
       @Override
-      public Weight createWeight(IndexSearcher searcher, ScoreMode scoreMode, float boost) {
-        return new Weight(this) {
-
-          @Override
-          public Explanation explain(LeafReaderContext context, int doc) {
-              return Explanation.match(0f, "No match on id " + doc);
-          }
-
-          @Override
-          public Scorer scorer(LeafReaderContext leafReaderContext) {
-            return null;
-          }
-
-          @Override
-          public boolean isCacheable(LeafReaderContext ctx) {
-            return false;
-          }
-        };
+      public Weight createWeight(IndexSearcher searcher, ScoreMode scoreMode, float boost) throws IOException {
```

Comment:
Do we need this method at all?

Reply:
Actually that's right, we shouldn't need it anymore, it's basically pointless given the changes we have in DocSetQuery. I'll run the tests to confirm. Thanks!